conf/threat_intel.json

```diff
@@ -0,0 +1,10 @@
+{
+  "enabled": false,
+  "mapping": {
+    "destinationAddress": "ip",
+    "destinationDomain": "domain",
+    "deviceAddress": "ip",
+    "fileHash": "md5",
```

Comment:
is `fileHash` always an MD5, or could it be a SHA1 or SHA256 hash?

Reply:
Yes, it can be a SHA1 or SHA256, and `fileHash` is defined in [conf/types.com](https://github.com/airbnb/streamalert/blob/master/conf/types.json#L50). The IOC source uses `md5` designation for SHA and MD5.

Reply:
`The IOC source uses md5 designation for SHA and MD5.` - What do you mean, I don't follow :)

Reply:
I will do move `enable_threat_intel` parameter into rule processor settings in each cluster right now. So that Threat Intel can be toggled per cluster based.